Library/Formula/ansible.rb

```diff
@@ -22,6 +22,7 @@ class Ansible < Formula
   depends_on :python if MacOS.version <= :snow_leopard
   depends_on "libyaml"
   depends_on "openssl"
+  depends_on "libffi" if OS.linux?
```

Comment:
Digging a bit more.  I've made the change to use `unless OS.mac?`.  I'm still having trouble.

I think that the problem is that the *libffi* formula does not install *ffi.h` into the include directory, so building cffi fails.

*Should* the *libffi* formula copy/link/... *ffi.h*?